server/monitor/src/main/java/org/apache/accumulo/monitor/rest/manager/ManagerResource.java

```diff
@@ -79,11 +79,12 @@ public static ManagerInformation getTables(Monitor monitor) {
       String label = "";
       if (gcStatusObj != null) {
         long start = 0;
-        if (gcStatusObj.current.started != 0 || gcStatusObj.currentLog.started != 0) {
-          start = Math.max(gcStatusObj.current.started, gcStatusObj.currentLog.started);
+        if (gcStatusObj.getCurrent().isSetStarted() || gcStatusObj.getCurrentLog().isSetStarted()) {
```

Comment:
I think `isSet` would be equivalent to `!= 0` for these cases since the variable is a `long` but usually `isSet` checks if it is `null` i think.

Reply:
Primitives can't be null. So a value of `0` could mean it was never set, or it could mean that it was set explicitly to `0`. With `isSet...()` we can tell the difference, because Thrift serializes extra information about whether the `set...()` method was called. If we don't use that, we can't tell the difference. That difference may or may not matter, depending. In some `if` conditions, we want `is not set or is set to 0` vs. `is set to non-zero` and in other cases we want `is not set` vs. `is set to anything`.

I'm not sure which is the case here.